pcsd/locales/en-US.yml

```diff
@@ -0,0 +1,192 @@
+    resourceSet: "Enter the resources you want to be in one set into the 'Set' field separated by space.Use the 'New Set' button to create more sets.Use the 'Add' button to submit the form."
+    reaourceCreatingInfo: "Do not start the resource automatically after creating."
+    nodeLinksHelper: "Please specify the count of links that is compatible with this cluster"
+    clusterInfoHelper: "Select a cluster to view more detailed cluster information"
+    clusterNameHelper: "Enter the name of a node in a cluster that you would like to manage"
+    clusterNodeNameHelper: "Enter the names of the nodes you would like to use to create a cluster."
+    selectTransportType: "Select transport type"
+    compressionHelper: "Compression is supported only with knet."
+    encryptionHelper: "Encryption is supported only with knet."
+    thereAreUnsavedChangesInTheForm: "There are unsaved changes in the form."
+    sessionExpiredHelper: "Your session has expired. Log in again, please."
+    authenticateHelper: "Enter password for user 'hacluster' to authenticate nodes."
+    useSamePasswordForAllNodes: "Use same password for all nodes"
+    useCustomAddressAndPort: "Use custom address(es) and port(s)"
+    removeClusterConfirm: "Are you sure you want to remove the following cluster(s) from the GUI? (This only removes the cluster from the GUI, it does not stop the cluster from running.)"
+    destroyClusterConfirm: "Are you sure you want to destroy the following cluster?"
+    destroyClusterConfirmExtraMessage: "The cluster will be stopped and all its configuration files will be deleted. This action cannot be undone."
+    removeNodeConfirm: "Are you sure you want to remove the following node(s)?"
+    removeRescourceConfirm: "Are you sure you want to remove the following resource(s)?"
+    EnforceRemoveRescource: "Enforce removal (Remove the resources without stopping them first.)"
+    removeACLConfirm: "Are you sure you want to remove the following ACL role(s)"
+    disableSBDConfirm: "Are you sure you want to disable SBD?"
+    issueFound: "Issue Found"
+    readDescription: "Allows to view cluster settings"
+    writeDescription: "Allows to modify cluster settings except permissions and ACLs"
+    grantDescription: "Allows to modify cluster permissions and ACLs"
+    fullDescription: "Allows unrestricted access to a cluster including adding and removing nodes and access to keys and certificates"
+    resourceSelectGroupHelper: "Select a group to add the resource to."
+    clusterHasBeenCreated: "Cluster has been succesfully created."
+    startCluster: "Start the Cluster"
+    displayAll: "displaying all"
+    displayIssue: "displaying only issues"
+    clusterNameAlreadyUsed: "The cluster name '%{clusterName}' has already been added. You may not add two clusters with the same name."
+    nodeNameAlreadyUsed: "The node '%{node_name}' is already a part of the '%{cluster_name}' cluster. You may not add a node to two different clusters."
```

Comment:
In string variables `node_name` and `cluster_name` do not conform with parameters used in code.

I feel like usage of i18n is especially error-prone in cases like this (typos in translation keys etc.) and we will find such issues during runtime. Therefore I would prefer to have some checks to prevent these mistakes. After a quick search I found https://github.com/glebm/i18n-tasks
Could integrate it into our test suite?